home-game/home-game.html

```diff
@@ -1,13 +1,44 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
   <link rel="stylesheet" href="home-game.css" />
-  <title>GAMSIC</title>
+  <title>Document</title>
 </head>
 <body>
-  <script src="home-game.js"></script>
+<!-- start header -->
+<header>
+  <i class="ri-arrow-right-s-line"></i>
+  <i class="ri-sun-fill"></i>
+</header>
+<!-- end header -->
+<main>
+  <section class="game-img">
+    <img src="../assets/images/Xbox-wireless-controller-carbon-black%202.png">
+  </section>
+
+  <section class="game_description">
+    <section class="title">
+        <h1 class="game-title">GAMSIC</h1>
+        <h6 class="game-subtitle">GAMING</h6>
+    </section>
+
+  <section class="description">
+    <p >a web page that displays product details about the game console.</p>
+  </section>
+
+    <section class="game-button">
+      <button class="btn"> <a href="#"> Learn More </a> </button>
```

Comment:
You should link the button to the next page when the user wants to go for further information.